src/services/movie-search.ts

```diff
@@ -0,0 +1,36 @@
+import { API_KEY, getFilmsUrl } from "../constants/constants";
+import { movieType } from "../enums/enum";
+export class MovieService {
+    public async getMovies(item?: string) {
```

Comment:
Again it doesn't make sense to combine async await and then in one function. You need to choose one of them